game/lobby.go

```diff
@@ -11,6 +11,11 @@ import (
 	"strings"
 	"sync"
 	"time"
+	"unicode"
+
+	"golang.org/x/text/runes"
```

Comment:
It appears you aren't using a formatter. What editor are you coding with? Most well known editors have some kind of golang support, maybe you should check it out.

Reply:
I'm using VS Code, do you say that about runes import?
When I compile with just "runes" instead, I get an error saying it's undefined. I don't get everything with Go package, and what is standard or not, and that stuff.

Reply:
What I mean is, that there's a Go extension (plugin) for VS-Code, just make sure you got that installed, because your imports had difference spacing in front of them, indicating that you didn't use a formatter. No biggie though, just wanted to mention it